apps/example-tailwind/src/app/pages/page-profile/page-profile.component.html

```diff
@@ -1,25 +1,712 @@
+                                        >
+                                        <input
+                                            class="h-8 w-full rounded-md border-[1.5px] p-2 placeholder:text-sm"
+                                            id="phone"
+                                            type="tel"
+                                            pInputText
+                                            name="phone"
+                                            [(ngModel)]="user.phone"
+                                            [disabled]="isPersonalInfoDisabled"
+                                            [placeholder]="user.phone" />
+                                    </div>
+                                </div>
+
+                                <div class="flex justify-between gap-5">
+                                    <div class="p-field w-1/2">
+                                        <label
+                                            for="birthDate"
+                                            class="mb-1 block text-sm font-semibold"
+                                            >Birth Date</label
+                                        >
+                                        <p-calendar
+                                            styleClass="!h-8 !rounded-md !placeholder:text-sm w-full"
+                                            id="birthDate"
+                                            name="birthDate"
+                                            [(ngModel)]="user.birthDate"
+                                            [disabled]="isPersonalInfoDisabled"
+                                            [showIcon]="true"
+                                            dateFormat="mm/dd/yy"
+                                            [placeholder]="
+                                                (user?.birthDate
+                                                    | date: 'MM/dd/yyyy') ?? ''
+                                            "></p-calendar>
+                                    </div>
+                                    <div class="w-1/2"></div>
+                                </div>
+
+                                <div class="p-field mt-2 flex justify-end">
+                                    <button
+                                        type="submit"
+                                        pButton
+                                        label="Save"
+                                        class="bg-green ml-5 rounded-sm px-4 py-1 text-sm text-white"
+                                        *ngIf="
+                                            !isPersonalInfoDisabled
+                                        "></button>
+                                </div>
+                            </form>
+                        </div>
+                    </p-card>
+                </div>
+            </div>
+
+            <div class="">
+                <div>
```

Comment:
please remove  empty class and extra div
```suggestion
            <div>
```